### ISSUE TITLE
Thonny stops responding while installing package

### ISSUE THREAD

**Issue by JourdanFleurus** [thonny/thonny#1855]
Hello,

Today I installed Thonny on my Windows 10 machine and started installing packages from `Tools > Manage Packages`. However, in the middle of installing `cvxpy`, Thonny stopped responding and I had to force quit the program. On the next attempt, Thonny didn't crash, but I received  this error message:  
![1](https://user-images.githubusercontent.com/85505571/121044006-27f5de80-c783-11eb-8b67-505d43d45878.png)
It tells me to open the logs for full command output, but I'm not sure how to do that.

Then I restarted my computer, and on the third attempt, Thonny crashed again. This time I paid more attention and noticed that the error occured while it was "installing build dependencies":

![2](https://user-images.githubusercontent.com/85505571/121045180-b2d6d900-c783-11eb-8da7-8d28a6143c08.png)
In case it's relevant: some time ago I downloaded Python directly from the official website, and I also have PyCharm installed.
After the failures with `cvxpy` I tried to install `pillow`, and I was succesful.

I would greatly appreciate any help with this issue. Thank you in advance!

**Comment by aivarannamaa** [comment]
Thank you for the report! I'll try to reproduce it soon.

**Comment by JourdanFleurus** [comment]
Dear aivarannamaa,

Thank you for your interest! I ended up (half) figuring out what the problem was: I did not have any of the dependencies installed, as I didn't know that dependencies were a thing (yes I'm VERY green haha). Now when I tried to install the dependencies, I did run into some errors. 

How I ended up getting cvxpy up and running was: I think I changed Thonny from using Python 3.7 to Python 3.9. And the Python  3.9 folder had all the packages I wanted, since I installed them earlier with PyCharm (which automatically installs all dependencies I guess).
